src/components/Label/README.md

```diff
@@ -275,3 +297,4 @@ LANDING_BLOCK-->
 | value            | Label value (displayed as `"children : value"`) |                `string`                 |             |
 | title            | `title` HTML attribute                          |                `string`                 |             |
 | qa               | `data-qa` HTML attribute, used for testing      |                `string`                 |             |
+| animation        | Animation type to apply to the label            |              `'gradient'`               |             |
```

Comment:
Update according to prop name change

Reply:
Sorry, forgot to fix. Fixed here https://github.com/gravity-ui/uikit/pull/2297/commits/8db583d6d261c3991c8307a053ae54e7c8a037b4